src/Controls/src/Core/Handlers/Items/Android/Adapters/ItemsViewAdapter.cs

```diff
@@ -43,11 +43,7 @@ protected virtual TItemsViewSource CreateItemsSource()
 
 		protected virtual void ItemsViewPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs property)
 		{
-			if (property.Is(Microsoft.Maui.Controls.ItemsView.ItemsSourceProperty))
```

Comment:
Why are we no longer updating the ItemsSource when it changes?

Reply:
The mapper does this